src/server.js

```diff
@@ -3,11 +3,31 @@ const { Command } = require("commander");
 const path = require("path");
 const fs = require("fs");
 const { Groq } = require("groq-sdk");
+const toml = require("smol-toml");
 require("dotenv").config();
 const promptAI = require("./ai");
 const defaultPrompt = require("./defaultPrompt");
 const getFileContent = require("./getFileContent");
 
+/**
+ * Parse data from a toml configuration file
+ * @param {string} filepath
+ * @returns parsed configuration data
+ * @throws Error if parsing operation fails
+ */
+function loadConfig(filepath) {
+  try {
+    const data = fs.readFileSync(filepath, "utf-8");
+    return toml.parse(data);
+  } catch (err) {
+    // Output error except non-existent file error
+    if (err.code !== "ENOENT") console.error("Error:", err);
+  }
+}
+
+// Load configuration from config.toml
+const config = loadConfig("../config.toml");
```

Comment:
I had an error when I was trying to run this code in my machine, I think you should resolve the file first to always get the absolute path:
```
// Resolve the path to the configuration file
const configPath = path.resolve(__dirname, "../.toml");

// Load configuration from config.toml
const config = loadConfig(configPath);
```

Reply:
And because now you are reading config from toml file. But there are some cases as:
- If the file is present but can't be parsed as TOML, exit with an appropriate error message.
- And to make the code more concise, you can create a `finalConfig` variable to get the final results for all options then only call `promptAI ()` function one time:
```js
// Final configuration
    const finalConfig = {
      model: options.model || config.model,
      temperature: options.temperature || config.temperature,
    };
    console.log(`temperature: ${finalConfig.temperature}`);
    console.log(`model: ${finalConfig.model}`);

    // Combine data if user input multiple files
    const outputData = getFileContent(files, options);
    promptAI(defaultPrompt + outputData, finalConfig.temperature, options);
```